src/components/filterbox.jsx

```diff
@@ -56,59 +100,67 @@ export class Filterbox extends React.Component {
     }
   }
 
-  toggleValue(event, item) {
-    event.stopPropagation();
+  onRowClick({ rowData }) {
     const { model } = this.props;
-    model.selectListObjectValues('/qListObjectDef', [item.qElemNumber], true);
+    if (rowData) {
+      if (rowData[0].qState !== 'S') {
+        // eslint-disable-next-line no-param-reassign
```

Comment:
IMO, eslint rules exist for some reasons :)
Use a let for assigning, and rowData[0].qState = let variable